tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = ui-tests
+skipsdist = True
+
+[testenv]
+passenv = *
+
+[testenv:ui-tests]
+commands =
+    pip install -r tests/integration/requirements.txt
+    pytest --driver Firefox tests/integration/ {posargs}
+
+[testenv:formatting]
```

Comment:
you should remove this in favor of the `.therapist.yml` changes mentioned above.